packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -49,29 +49,16 @@ export class GalleryContainer extends React.Component {
     };
     this.videoScrollHelper = new VideoScrollHelper(videoScrollHelperConfig);
 
-    if (utils.isSSR()) {
```

Comment:
This needs to be done in the wrapper.
Saving the SSR state so that the client can start from there